views/doc.list.html.twig

```diff
@@ -3,36 +3,50 @@
 {% import _self as macros %}
 
 {% block content %}
-    <div id="searchbar" class="clearfix">
-        <input id="docsearch" type="text" placeholder="Search documentation">
-    </div>
-
-    <h1>Book</h1>
-    <ul>
-    {% for filename, data in book %}
-        {{ macros.renderFile(filename, data) }}
-    {% endfor %}
-    </ul>
-    <h1>Articles</h1>
-    <ul>
-    {% for filename, data in articles %}
-        {{ macros.renderFile(filename, data) }}
-    {% endfor %}
-    </ul>
-    <h1>FAQs</h1>
-    <ul>
-    {% for filename, data in faqs %}
-        {{ macros.renderFile(filename, data) }}
-    {% endfor %}
-    </ul>
-
-    <h1>API Docs</h1>
-    <p>If you need to work with composer as a library you may also want to check out the <a href="/apidoc/master/index.html">API Docs</a></p>
+    <section role="main">
+
+        <div id="searchbar" class="clearfix">
+            <input id="docsearch" type="text" placeholder="Type here to search documentation..." role="searchbox"
+                   title="Enter text here and submit to search the documentation"/>
+        </div>
+
+        <h1 role="sectionhead">📚 Book</h1>
```

Comment:
Not sure what icon you used here, but my default font does not support it.

Reply:
Interesting. Those are emojis, they're not universal. 
What's your platform? I might just remove them. Just thought this would add some visual landmarks on page, they're far from mandatory :)

Reply:
I'm on Linux.

Reply:
Thanks. What would say in % emojis you see appear on this page?
https://getemoji.com/

Reply:
In Chromium I see about 10%. In Firefox I see everything. But I believe that is because one of the fonts defined for use is "Twemoji Mozilla" (and I can see that that font is used to render them in Firefox).

Reply:
After installing the noto emoji font from google, all emojis work in Chromium too. But kind of iffy to rely on emoji fonts imho. Better to use something fontawesome.

Reply:
Well, emojis are supposed to be universal, in the end. They're usually rendered by the device itself.
Since this is made for accessibility, if it's not, I might as well remove them. 

Thanks a lot for the feedback! :)

Reply:
Closed: removed the emojis. :)